src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -160,6 +161,12 @@
        :accessibility-label :about-button
        :chevron             true
        :on-press            #(re-frame/dispatch [:navigate-to :about-app])}]
+     (when local-pairing-mode-enabled? [quo/list-item
+                                        {:icon                :main-icons/mobile
```

Comment:
why `main-icons` is that not from the old set? should we not update to new set?

Reply:
hmm this was made 3 months ago, the idea was to swiftly get the UI up and running, I shall refactor :)

Reply:
Hmm now that I think of it, all the icons in the list have old icons here, so it makes sense to keep it the way it is for now, whoever will build the new screen for profile settings will modify this icon and that will look better with new UI 
ref : https://www.figma.com/file/aztkse2iYuMAjVJRAsxqFO/Syncing-Settings?node-id=1%3A39597&t=tlPL26WdJQxuqA4b-0